skrub/_reporting/_table_report.py

```diff
@@ -19,8 +19,8 @@ class TableReport:
 
     Parameters
     ----------
-    dataframe : pandas or polars DataFrame
-        The dataframe to summarize.
+    dataframe : pandas or polars Series or DataFrame.
```

Comment:
Numpydoc raises an error because of the final dot:
```suggestion
    dataframe : pandas or polars Series or DataFrame
```